test/conformance/CMakeLists.txt

```diff
@@ -109,6 +109,7 @@ add_subdirectory(platform)
 add_subdirectory(device)
 add_subdirectory(context)
 add_subdirectory(memory)
+add_subdirectory(memory-migrate)
```

Comment:
I think this should move down to line 140 along with all the other test suites that depend on kernel execution.

Reply:
OK sure, have moved.